spec/system/order_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Order check', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let!(:employee) { create(:employee) }
+  let!(:company_value) { create(:company_value) }
+
+  it 'Buying a reward' do
+    create(:kudo, giver: employee, receiver: employee, company_value: company_value)
```

Comment:
Your factory will create `company_value` for you, and can also create another kudo-giver `employee`, so the kudo creation can look like this:

```suggestion
    create(:kudo, receiver: employee)
```

Reply:
You're right, I should use the factory if I can :)